Added a note in an attempt to avoid confusion (#3972)
umbraco/UmbracoDocs: Reference/Querying/MemberShipHelper/index.md

---
versionFrom: 8.0.0

> versionTo: 8.0.0

Reviewer: I am just wondering if we add this then the version box would show as 8.0.0 - 8.0.0 (current) which will make it question if the article is not valid for 8+versions.

Maybe add versionTo: 8.18.4?

Author: That's also an option.
Then perhaps let's do 8.18.0 then?
That way, if we release patches then we won't have to go in an update this one every time? 🙈

Reviewer: Yup, sounds good 💪

> versionTo: 8.0.0

Reviewer: I am just wondering if we add this then the version box would show as 8.0.0 - 8.0.0 (current) which will make it question if the article is not valid for 8+versions.

Maybe add versionTo: 8.18.4?

versionRemoved: 9.0.0
---

:::warning
The topic covered in this article is only relevant to you if you are using Umbraco 8 or below.

> This article is not relevant to you if you are using Umbraco 9 or above.

Reviewer: ```suggestion
If you are using Umbraco 9 or above, see the [IMemberManager](../IMemberManager/index.md) article.
```

> This article is not relevant to you if you are using Umbraco 9 or above.

Reviewer: ```suggestion
If you are using Umbraco 9 or above, see the [IMemberManager](../IMemberManager/index.md) article.
```
```suggestion
This article is not relevant to you if you are using Umbraco 9 or above.
```
